frontend/src/components/cluster/Overview.tsx

```diff
@@ -116,15 +116,44 @@ function EventsSection() {
         columns={events ? [
           {
             label: 'Type',
-            getter: event => event.involvedObject.kind
+            getter: event => event.involvedObject.kind,
+            sort: (e1: Event, e2: Event) => {
+              if (e1.involvedObject.kind < e2.involvedObject.kind) {
+                return 1;
+              } else if (e1.involvedObject.kind > e2.involvedObject.kind) {
+                return -1;
+              } else {
+                return 0;
+              }
+            }
```

Comment:
This is the thing we can easily make automatic as written in my previous patch's comments.